CredentialProvider.Microsoft/CredentialProviders/Vsts/IAuthUtil.cs

```diff
@@ -44,6 +46,19 @@ public AuthUtil(ILogger logger)
             this.logger = logger;
         }
 
+        public async Task<string> GetTenantIdAsync(Uri uri, CancellationToken cancellationToken) 
```

Comment:
let's discuss this one in person.. the tenant is part of the endpoint authorization which is the "official" place, and extracted from there in other places. Not opposed to the resource tenant one if that's officially supported. In either case, we really want to pull 2 values at the same time and pass them around, so should consider a struct or something to contain this info (and update other MSAL stuff to use the new TenantId stuff). Oh, and test for MSAs as well.

Reply:
sounds good - I thought the auth endpoint can have non-tenant id information such as /organization

Reply:
moved to struct

Reply:
resolving - reopen or let me know if you have any feedback on the implementation